lib/Dialect/FIRRTL/LowerToRTL.cpp

```diff
@@ -97,6 +97,23 @@ static LogicalResult lower(firrtl::ConstantOp op, ArrayRef<Value> operands,
   return success();
 }
 
+static LogicalResult lower(firrtl::WireOp op, ArrayRef<Value> operands,
+                           ConversionPatternRewriter &rewriter) {
+  auto resType = op.result().getType().cast<FIRRTLType>();
+  if (auto resultType = RTLTypeConverter::convertType(resType)) {
+
+    if (auto intType = resultType.getValue().dyn_cast<IntegerType>()) {
+      rewriter.replaceOpWithNewOp<rtl::WireOp>(op, intType, op.nameAttr());
```

Comment:
I don't think this 'if' is actually necessary.  When convertType starts supporting other cases, they should "just work" here, so I think you can drop the test and just pass `resultType.getValue()` into the creation of the `WireOp`.